src/components/Header/Header.jsx

```diff
@@ -1,18 +1,32 @@
-import React from 'react';
+import React, { useState } from "react";
 import { Link } from 'react-router-dom';
-import styles from './Header.module.css';  
+import icon_close from "../../assets/icon-close.svg";
+import logo from "../../assets/logo.svg";
+import styles from "./Header.module.css";
 
 const Header = () => {
+  const [mobileMenu, setMobileMenu] = useState(false);
+  const toggleMenu = () => {
+    mobileMenu ? setMobileMenu(false) :
+   setMobileMenu(true);
+  };
   return (
+  
     <header className={styles.header}>
-      <nav>
-        <ul>
-          <li><Link to="/">Home</Link></li>
-          <li><Link to="/about">About</Link></li>
-          <li><Link to="/contact">Contact Us</Link></li>
+      <img src={logo} alt="myteam logo" className={styles.logo} />
+      <nav className={styles.header-nav}>
+        <ul className={ mobileMenu ? '' :  styles['hide-mobile-menu']}>
+          <img src={icon_close} alt="" className={styles.icon_close} onClick={toggleMenu} />
+       <li className={styles.nav-list-item}><Link to="/">Home</Link></li>
+          <li className={styles.nav-list-item}><Link to="/about">About</Link></li>
```

Comment:
same issue as above with hyphenated class name syntax

please also fix lines 20 and 22 to align correctly with the indentation level of line 21